### PR TITLE
Added Korean translation

### PR DESCRIPTION
**Pull request by pine5508**
owo!

### CODE REVIEW

File: translations/ko.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "error-occured": "오류가 발생하였습니다: ",
+    "version-update-message": "ezpp가 업데이트 되었습니다! v{0} 실행 중",
+    "changelog": "수정사항",
+    "accuracy": "정확도",
+    "combo": "콤보",
+    "empty-fc": "(비어 있음 = FC)",
+    "misses": "미스",
+    "mod-hidden": "히든",
```
Comment (oamaok) [inline translations/ko.json]: just to make sure, are these comonly used names for the mods? Korean would be the only language to use language-specific mod names.

**Comment by oamaok** [comment]
uwu

File: translations/ko.json (hunk 1 above)
Comment (pine5508) [reply translations/ko.json]: Usually they use abbreviations such as "full combo" -> "ful com" etc, or just convert pronunciation of English into Hangul (Korean Alphabet). In formal-ish situations such as in an app, on a website etc, the use of abbreviations is not recommended.

Comment (oamaok) [reply translations/ko.json]: okay, thank you very much for the clarification!

**Comment by oamaok** [comment]
looking good, thank you for your contribution!

**Pull request merged**